### ISSUE TITLE
Suggest Ideas

### ISSUE THREAD

**Issue by PrinceZaman-2005** [PrinceZaman-2005/DraftHead#2]
Hey dude, Here you see this model overfits too much and only make ~64% on test set.
I can't make it push more, I tried adding better logits, reducing drafts, params and epochs but results is still moderate.
I know you have better ideas but i would like you to conduct experiments too! 
You see i can't experiment so many ideas.

> Let's make this happen together

**Comment by HeroSonic-123** [comment]
Dude add drop-out to prevent overfitting
